src/BizHawk.Emulation.Cores/Consoles/Sony/PSP/PPSSPP.IStatable.cs

```diff
@@ -0,0 +1,56 @@
+using System.IO;
+
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Consoles.Sony.PSP
+{
+	public partial class PPSSPP : IStatable
+	{
+		private byte[] _stateBuf = Array.Empty<byte>();
+
+		public bool AvoidRewind => true;
+
+		public void SaveStateBinary(BinaryWriter writer)
+		{
+
+			var stateLen = _libPPSSPP.GetStateSize();
+			writer.Write(stateLen);
+
+			if (stateLen > _stateBuf.Length)

```

Comment:
Should this be `!=`? Under what circumstances could this happen?

Reply:
fixed